build

```diff
@@ -2,25 +2,28 @@ set -x
 path=$1
 if [ "$path" = "" ]
 then
-    echo Usage: ./build \<path to netvirt\>
+    echo Usage: ./build \<path-to-repo\> [feature-list]
     exit 2
 fi
 targetpath=${path}/karaf/target
-feature="odl-netvirt-openstack"
+feature=${2:-"odl-netvirt-openstack"}
```

Comment:
To me it makes sense that if the user passes in a list of features that should be it, no? It would be more flexible. Like, maybe someone would want to use this to test some feature that has nothing to do with netvirt or genius, etc. On the other hand, if someone is specifying the list of features, it's not such a big deal to tack on odl-netvirt-openstack.